sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -29,50 +29,79 @@ In order to export telemetry data to Azure Monitor, you will need the instrument
 search for your resource. On the overview page of your resource, you will find the instrumentation key in the top
 right corner.
 
-### Creating exporter builder for Azure Monitor
-```java readme-sample-createExporterBuilder
-AzureMonitorExporterBuilder azureMonitorExporterBuilder = new AzureMonitorExporterBuilder()
-    .connectionString("{connection-string}");
+
+### Setup the OpenTelemetry SDK to work with Azure Monitor exporter
+
+The following code shows how to configure the OpenTelemetry SDK auto-configuration with the Azure Monitor exporter:
+```java readme-sample-autoconfigure
+AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
+new AzureMonitor("{connection-string}")
```

Comment:
it looks weird to create an instance and ignore it. Could we consider API like 

```java
sdkBuilder = AzureMonitor.configure("{connection-string}", sdkBuilder);
```

or 

```java
OpenTelemetry openTelemetry = new AzureMonitor("{connection-string}")
   .configure(sdkBuilder) // or configureAndBuild() if it makes sense
   .build()
   .getOpenTelemetrySdk()
```